docs/user-guide.md

```diff
@@ -0,0 +1,6 @@
+# Concepts
+
+Imagine a world without kittens. What a world that would be.
```

Comment:
okay seriously, last night we were playing the expansion to Betrayal at the House on the Hill, and the event was literally, you have to soothe kittens while preparing cat food for the large cat.